src/TodoListItem.js

```diff
@@ -1,7 +1,19 @@
 import React from "react";
 
-const TodoListItem = ({ item }) => {
-  return <li>{item.title}</li>;
+const TodoListItem = (props) => {
+  const {title, id, onRemoveTodo} = props;
+  const handleRemoveClick = () => {
+    onRemoveTodo(id);
+  };
+
+  return (
+    <li>
+      <span>{title}</span>
+      <button type="button" onClick={handleRemoveClick}>
+        Remove
+      </button>
```

Comment:
Related to my previous comment, you don't need to create another function that only calls `onRemoveTodo`, you can use it directly in `onClick`
```suggestion
      <button type="button" onClick={() => onRemoveTodo(id)}>
        Remove
      </button>
```